tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -67,6 +68,7 @@ static void unfollow_during_transform (GumStalkerIterator * iterator,
 static gboolean store_range_of_test_runner (const GumModuleDetails * details,
     gpointer user_data);
 static void pretend_workload (GumMemoryRange * runner_range);
+static gpointer increment_integer (gpointer data);
```

Comment:
Should be in chronological order. (Remember to update this after moving the function re. comment below.)

Reply:
Done (at least I think so)